Integration portal GA (#930)
amplitude/amplitude-docs: content/collections/partners/en/create-a-cohort-syncing-integration.md

### Preview & test endpoint

Before submitting your configuration for review, test the mock payload that you expect to receive from Amplitude. In Testing Tab, follow these steps to preview and test your configuration.

In the Testing tab, the Destination Settings is identical to what users of your integration will see. This form is used to control the parameters you have defined in the headers of the payload. 

> To generate users, you can upload a CSV or click regenerate which will generate users based on your configurations. The CSV must have an Operation column which must be either add or remove. Any parameters used in the List Creation payload must also be the same across all rows in the CSV. The CSV must also have columns for each of the parameters used. 

Reviewer: ```suggestion
To generate users, upload a CSV or click **Regenerate** which generates users based on your configurations. The CSV must have an `Operation` column with values of either `add` or `remove`. Any parameters in the List Creation payload must be the same across all rows in the CSV. The CSV must have columns for each of the parameters used. 
```

> Here is a sample CSV that can be used for the default configurations: 

Reviewer: ```suggestion
Here is a sample CSV you can use for the default configurations: 
```

Use the `$list_id` you retrieved to test the add to add users and remove users endpoints.

> You can also test end to end with the **Test Endpoint** button in the Test Integration section, which will automatically run all necessary tests. 

Reviewer: ```suggestion
Click **Test Endpoint** in the Test Integration section to start an end-to-end test, which runs all necessary tests. 
```

## Release Internally

> To test in your org, click **Release Internally**. This enables anyone within your org to use the integration you defined. The integration should be instantly available once you do this. 

Reviewer: ```suggestion
To test in your org, click **Release Internally**. This enables anyone within your org to use the integration you defined. The integration should be instantly available after you do this. 
```
